examples/ensemble/plot_gradient_boosting_regression.py

```diff
@@ -19,38 +25,81 @@
 
 from sklearn import ensemble
 from sklearn import datasets
-from sklearn.utils import shuffle
 from sklearn.metrics import mean_squared_error
+from sklearn.model_selection import train_test_split
+
+##############################################################################
+# Load the data
+# -------------------------------------
+#
+# First we need to load the data. We will work on both datasets
+# simultaneously. We set random state to be consistent with the result.
```

Comment:
I find this phrasing weird. In the scikit-learn project the breast cancer dataset is considered a single dataset with two components: input and target variables.